doc/release/README.tasks.rst

```diff
@@ -2,6 +2,12 @@
 Chapel programs, tasks, and threads
```

Comment:
Done.

On Sep 21, 2015, at 11:16 AM, Elliot Ronaghan <notifications@github.com<mailto:notifications@github.com>> wrote:


In doc/release/README.tasks.rst<https://github.com/chapel-lang/chapel/pull/2593#discussion_r40005462>:

> @@ -2,6 +2,12 @@
>  Chapel programs, tasks, and threads


Mind changing this to just "Chapel Tasks"?

—
Reply to this email directly or view it on GitHub<https://github.com/chapel-lang/chapel/pull/2593/files#r40005462>.